src/__tests__/views/commentView/__test__/commentView.test.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { mount } from "enzyme";
+import { CommentView } from "../../../../views/commentView";
+
+describe("CommentView", () => {
+    const props = {
+       event: {
```

Comment:
I think it would be wise to add a peventDefault function mock here since you are doing a submit

Reply:
It has been added thank you for that